File: python/tictactoe.py
class TicTacToeSolver:
    def __init__(self):
        self.board = [[' ' for _ in range(3)] for _ in range(3)]

    def print_board(self):
        for row in self.board:
            print('|'.join(row))
            print('-' * 5)

    # TODO: Implement the rest of the game

File: go/tictactoe.go
package main

var board [3][3]int

func PrintBoard() {
	for i := 0; i < 3; i++ {
		for j := 0; j < 3; j++ {
			print(board[i][j])
		}
		println()
	}
}

// TODO: Implement the rest of the game